patches/chrome-installer-mac-BUILD.gn.patch

```diff
@@ -1,11 +1,13 @@
 diff --git a/chrome/installer/mac/BUILD.gn b/chrome/installer/mac/BUILD.gn
-index b3fc448575b68fbde67a57040896f88fb77cf54c..7be27ac9fd258c15f14d8805b1b3fee2134374af 100644
+index b3fc448575b68fbde67a57040896f88fb77cf54c..22f09546d7edc2bf03679e9d2290f34c86b62b36 100644
 --- a/chrome/installer/mac/BUILD.gn
 +++ b/chrome/installer/mac/BUILD.gn
-@@ -18,6 +18,7 @@ group("mac") {
+@@ -17,7 +17,8 @@ group("mac") {
+   public_deps = [ ":copies" ]
  }
  
- _packaging_dir = "$root_out_dir/$chrome_product_full_name Packaging"
+-_packaging_dir = "$root_out_dir/$chrome_product_full_name Packaging"
++_packaging_dir = "$root_out_dir/" + string_replace("$chrome_product_full_name Packaging", " ", "_")
 +_packaging_dir = "$brave_packaging_base_dir/" + string_replace("$chrome_product_full_name Packaging", " ", "_")
```

Comment:
Curious about this file changes. Maybe changing above two lines(9 and 10) doesn't have any effect?

Reply:
have asked the same but @bridiver said it's the preferred way

Reply:
https://github.com/brave/brave-core/blob/1.18.x/patches/chrome-installer-mac-BUILD.gn.patch